.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,35 @@
+name: Pack and publish nugets
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build-pack:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install dotnet6
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: |
+            6.0.x
+      - name: Install deps
+        run: |
+          dotnet restore
+      - name: Build
+        run: |
+          dotnet build --configuration Release --no-restore -p:Deterministic=true -p:BuildNumber=${{ github.run_number }}
+      - name: Pack
+        run: |
+          dotnet pack AltinnIIClients.sln --configuration Release --no-restore --no-build -p:BuildNumber=${{ github.run_number }} -p:Deterministic=true
```

Comment:
in the case we have multiple projects in the solution, won't all projects be pushed with each release?

Reply:
For the test-project, IsPackable is set to false.
And we only push from src/**/bin/Release.